tests/queries/tests.py

```diff
@@ -2427,6 +2427,14 @@ def test_in_subquery(self):
             set(Eaten.objects.filter(food__in=Food.objects.filter(name='apple').values('eaten__meal'))),
             set()
         )
+        self.assertEqual(
+            set(Food.objects.filter(eaten__in=Eaten.objects.filter(meal='lunch'))),
+            {apple}
+        )
+        self.assertEqual(
+            set(Food.objects.filter(eaten__in=Eaten.objects.filter(meal='lunch').values('food__name'))),
+            set()
```

Comment:
Yes. It should. However so should the assert above this from the previous version as far as my knowledge of values works (IE. It shouldn't change the query).

As I believe that is yet a separate issue, I've left that test there and just copied it down for the exact reverse test as above.

I can remove it if you would prefer though. (Technically it is incorrect.)